.github/workflows/test.yml

```diff
@@ -174,26 +174,23 @@ jobs:
       - test_environment_linux
       - test_multiple_ros_installations
       - test_ros_binary_install_ubuntu
-    if: always() # run even if the dependent jobs have failed to log failures
-    # Allow build reports to fail on pull requests.
-    # When a contribution is made on a fork, the secrets will not be available,
-    # and this step will be failing. This is acceptable.
-    # On the other end, we want to be notified if this happens on merge, or
-    # on schedule.
-    continue-on-error: ${{ github.event_name == 'pull_request'}}
+    if: ${{ !cancelled() }} # Run even if the dependent jobs have failed, to log failures
     steps:
       - name: Configure AWS Credentials
         uses: aws-actions/configure-aws-credentials@v1
+        continue-on-error: ${{ github.repository != 'ros-tooling/setup-ros' }}
```

Comment:
Is this one needed? could you add a comment?

Reply:
It was, but not anymore since I just moved continue-on-error to the job level.